src/mmw/js/src/data_catalog/templates/form.html

```diff
@@ -1,6 +1,35 @@
-<i class="data-catalog-search-icon fa fa-search"></i>
-<input
-    class="data-catalog-search-input"
-    type="text"
-    placeholder="Search data sources"
-/>
+<button class="btn btn-link date-filter-toggle small">{{filterText}}</button>
+
+{% if showingFilters %}
+<div class="data-catalog-search-dates">
+    <div class="form-group">
+        <label>Before</label>
+        <input
+            class="data-catalog-date-input form-control"
+            data-date-range="start"
+            value="{{dateStart}}"
```

Comment:
Yeah, I found the labels problematic to map to the API attributes.  I'll assume they were determined casually and come up with something that can be better understood.